plugins/modules/network_compliance_workflow_manager.py

```diff
@@ -1259,37 +1028,43 @@ def get_task_result(self, task_id, device_ids):
         """
```

Comment:
This API also should be in dnac.py.. Can you please check? We already have APIs in dnac.py. If not, we have to move this API in dnac.py.

Reply:
Moved to dnac.py